modin/logging/config.py

```diff
@@ -31,6 +33,15 @@
 __LOGGER_CONFIGURED__: bool = False
 
 
+def _turn_off_logging():
+    """Turn off __LOGGER_CONFIGURED__."""
+    global __LOGGER_CONFIGURED__
+    __LOGGER_CONFIGURED__ = False
+
+
+atexit.register(_turn_off_logging)
```

Comment:
Shouldn't we add  processing of additional signals (ctrl+c, for example), as we made in unidist?:
https://github.com/modin-project/unidist/blob/81a355ac9a308aeb9b746a7e62859c4a097427eb/unidist/core/backends/mpi/core/executor.py#L879-L881

Reply:
I was able to fix the bug by making the thread daemonic instead of by registering a custom `atexit`. It turns out that that solution wasn't working, anyway.